source/solvers/vof.cc

```diff
@@ -628,10 +628,15 @@ VolumeOfFluid<dim>::finish_simulation()
         "error_phase", this->simulation_control->get_log_precision());
       this->error_table.write_text(std::cout);
     }
+
   if (this_mpi_process == 0 &&
-      this->simulation_parameters.multiphysics.vof_parameters.conservation
-          .verbosity == Parameters::Verbosity::extra_verbose)
+      this->simulation_parameters.post_processing.verbosity ==
+        Parameters::Verbosity::verbose &&
+      this->simulation_parameters.post_processing.calculate_mass_conservation)
     {
+      std::cout << "+------------------------------------------+" << std::endl;
+      std::cout << "|           VOF Mass Conservation          |" << std::endl;
+      std::cout << "+------------------------------------------+" << std::endl;
```

Comment:
There is a function to make this kind of header.
https://github.com/lethe-cfd/lethe/blob/master/include/core/utilities.h#L318

You could use that if you want, it's a bit more "regular"
If you want to amke one that makes table like this, we could also use taht.